server/services/caldav/api/caldav.controller.js

```diff
@@ -38,6 +38,16 @@ module.exports = function CalDAVController(caldavHandler) {
     });
   }
 
+  /**
+   * @api {patch} /api/v1/service/caldav/enable Enable or disable calendar synchronization
+   * @apiName Enable
+   * @apiGroup CalDAV
+   */
+  async function enable(req, res) {
```

Comment:
It's weird to have an "enable" function that either enable or disable

Either you have a PATCH calendar API call (simple), or either you have a "enable" + "disable" route.

Pick one :)

Reply:
I chose to split in 2 routes because we need actions after disabling